changelog.md

```diff
@@ -1,3 +1,6 @@
+# Future
+- [ADDED] Benchmarking feature (https://github.com/sequelize/sequelize/issues/2494)
```

Comment:
Change log should use links like [#2494](https://github.com/sequelize/sequelize/issues/2494)